### ISSUE TITLE
[a11y][TableSelection]: Number of selected checkbox item counts are not announced by screen-reader (JAWS)

### ISSUE THREAD

**Issue by alenvarg** [SAP/ui5-webcomponents#11482]
### Bug Description

When a checkbox item is selected, the screenreader announces it as selected, but the number of selected checkbox items is not announced by the screenreader. (JAWS)

Ensure that once we select the checkbox items, the number of selected items should be announced to screenreader users.
For example: 1 selected, 2 selected, 3 selected, etc..

### Expected Behaviour

Ensure that once we select the checkbox items, the number of selected items should be announced to screenreader users.
For example: 1 selected, 2 selected, 3 selected, etc..

### Isolated Example

https://sap.github.io/ui5-webcomponents/components/TableSelection/

### Steps to Reproduce

1. Scroll to Basic Sample
2. Select Multiple selection
3. Check different row items
4. The number of selected items should be announced to screenreader users. (JAWS)

![Image](https://github.com/user-attachments/assets/ff941d79-5024-493f-8674-9be1bcbb0ab0)

### Priority

Medium

### UI5 Web Components Version

2.9.0

### Browser

Edge, Chrome

### Organization

SAP

### Declaration

- [x] I’m not disclosing any internal or sensitive information.

**Comment by aborjinik** [comment]
I'm not aware of any ARIA specification/best-practice that advices such an announcement. This information isn't available to sighted users either.
If this is a specific requirement or personal preference, you might consider listening to the [change event of the selection component](https://sap.github.io/ui5-webcomponents/components/TableSelectionMulti/#change) and using an [invisible message](https://sap.github.io/ui5-webcomponents/nightly/docs/advanced/accessibility/#invisible-messaging)  to convey the information.

PS: For the reasons I explained above, I wouldn't recommend this.
